### PR TITLE
button styles / resource layout 

### PR DESCRIPTION
**Pull request by pegnott**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -41,6 +41,33 @@
             ]
           },
           "configurations": {
+            "staging": {
```
Comment (lukad03) [inline angular.json]: @pegnott It appears the staging configuration is defined twice. I would remove this from your PR. Otherwise you should be gtg

**Comment by lukad03** [comment]
Looks like there is a rebase conflict as well @pegnott . Keep the Slack button and remove the former link

File: angular.json (hunk 1 above)
Comment (pegnott) [reply angular.json]: updated to remove duplicate staging ref

**Comment by pegnott** [comment]
> Looks like there is a rebase conflict as well @pegnott . Keep the Slack button and remove the former link

conflict resolved with slack button in place of old content.

**Pull request merged**